### PR TITLE
Font should not change when a new theme is set

### PR DESCRIPTION
**Pull request by boutinb**
Fixes jasp-stats/jasp-test-release#1008

### CODE REVIEW

File: JASP-Desktop/qquick/jasptheme.cpp [hunk 1]
```diff
@@ -135,6 +135,9 @@ void JaspTheme::setCurrentTheme(JaspTheme * theme)
 	if(theme == _currentTheme)
 		return;
 
+	if (_currentTheme && theme)
+		theme->setFont(_currentTheme->font());
```
Comment (JorisGoosen) [inline JASP-Desktop/qquick/jasptheme.cpp]: Shouldn't the same happen for `_jaspCodeFont`?

Comment (JorisGoosen) [reply JASP-Desktop/qquick/jasptheme.cpp]: As in, the init does:
```
	setFont(						_jaspFont);
	setFontLabel(					_jaspFont);
	setFontRibbon(					_jaspFont);
	setFontGroupTitle(				_jaspFont);
	setFontPrefOptionsGroupTitle(	_jaspFont);
	setFontCode(					_jaspCodeFont);
	setFontRCode(					_jaspCodeFont);
```

So why not all of those here?

Comment (JorisGoosen) [reply JASP-Desktop/qquick/jasptheme.cpp]: Maybe it is better to make all the fonts static?
There not really supposed to be set from the theme now that we have font-settings right?

Comment (boutinb) [reply JASP-Desktop/qquick/jasptheme.cpp]: Yep, and my fix does not work well anyway...

**Comment by boutinb** [comment]
@JorisGoosen I have changed my fix

**Pull request merged**